add new rework phones alpha lab (#645)
pupil-labs/pupil-docs: alpha-lab/phone-neon/index.md

---
title: Uncover gaze behaviour on phone screens with Neon
description: "Evaluate Neon's accuracy on phone screens"
permalink: /alpha-lab/phone-neon
meta:
  - name: twitter:card
    content: player
  - name: twitter:image
    content: "https://i.ytimg.com/vi/gp5O1uskDME/maxresdefault.jpg"
  - name: twitter:player
    content: "https://www.youtube.com/embed/gp5O1uskDME"
  - name: twitter:width
    content: "1280"
  - name: twitter:height
    content: "720"
  - property: og:image
    content: "https://i.ytimg.com/vi/gp5O1uskDME/maxresdefault.jpg"
tags: [Neon, Cloud]
---

<script setup>
import TagLinks from '@components/TagLinks.vue'
</script>

# Uncover Gaze Behaviour on Phone Screens With Neon

<TagLinks :tags="$frontmatter.tags" />

<Youtube src="gp5O1uskDME"/>

> What catches your attention and how do you visually navigate through the interface/page of a mobile app? This article shows you the tools you can use to address these questions. 

Reviewer: ```suggestion
What catches your attention and how do you visually navigate through the interface of a mobile app? This article we use Neon, our wearable eye tracker, and all the other tools you need to address these questions. 
```

Author: Thanks. I've made a minor correction in the second sentence: "What catches your attention and how do you visually navigate through the interface of a mobile app? In this article, we use Neon, our wearable eye tracker, and all the other tools you need to address these questions."

Reviewer: sure, sounds good.

## Gaze Behaviour During Mobile App Engagement

> Researchers in fields such as UI/UX and neuromarketing frequently investigate user visual engagement with mobile applications. Wearable eye tracking is sometimes used for this purpose as it can provide valuable insights into user experience during mobile browsing. In this article, we used our latest wearable eye tracker, Neon, to capture and analyse users' viewing behaviour when focusing on small icons and features of mobile applications.

Reviewer: ```suggestion
UI/UX and neuromarketing researchers frequently investigate users visual engagement with mobile apps. Wearable eye tracking can provide valuable insights into user experience during mobile browsing. In this article, we used Neon, to capture and analyse users' viewing behaviour when focusing on small icons and features within a mobile app.
```

Reviewer: Do we want to note in the intro that we will be generating qualitative as well as quantitative analysis?

In this article, we test whether Neon, our latest eye tracker, can accurately capture and characterise viewing behaviour
while gazing at small icons on mobile applications. To achieve this, we used some of Alpha Lab's existing tutorials, including
how to generate scanpaths, define areas of interest (AOIs) and calculate outcome metrics, and map gaze onto dynamic content.
## Gaze Behaviour During Mobile App Engagement

Shall we start?
Researchers in fields such as UI/UX and neuromarketing frequently investigate user visual engagement with mobile applications. Wearable eye tracking is sometimes used for this purpose as it can provide valuable insights into user experience during mobile browsing. In this article, we used our latest wearable eye tracker, Neon, to capture and analyse users' viewing behaviour when focusing on small icons and features of mobile applications.

## What you'll need
## Overcoming Usability Challenges With Neon

Below you can find the tools we used for this project. Using these, you can replicate the content of this article with

> your own applications.

Reviewer: This section seems a little awkward IMO. I would consider omitting it or just condensing into a single sentence and putting into the previous section - `Gaze Behaviour During Mobile App Engagement`

> In order to characterise viewing behaviour on mobile phones using Neon, we first needed to make some eye tracking recordings whilst a user was engaging with a phone. We then leveraged Pupil Cloud and Alpha Lab tutorials to interpret the data and compute further metrics.

Reviewer: I feel like these two sentences might also be omitted, the steps below are quite clear IMO.

> Firstly, we placed the phone in a static position on a desk, allowing us to interact with the app.

Reviewer: Why does the phone need to be static? What is the motivation/constraint here? Reader needs to know that this is done so we can easily compare gaze data across recordings using RIM.

Author: Thanks for the feedback. I'll add an explanation: "1. We placed the phone in a static position on a desk, allowing us to interact with the app. This setup was crucial for utilizing our [Reference Image Mapper](https://docs.pupil-labs.com/neon/pupil-cloud/enrichments/reference-image-mapper/) enrichment which specifically requires the scene to have relatively static features in the environment."

Reviewer: Could also enable comparison across subjects right?

> Subsequently, we exported the data to analyse offline for a more in-depth exploration. For the offline analysis, we used existing Alpha Lab tutorials that allowed us to [generate scanpaths](/scanpath-rim/), [define areas of interest (AOIs) and calculate gaze metrics](/gaze-metrics-in-aois/), and [map gaze onto dynamic content](/map-your-gaze-to-a-2d-screen/).

Reviewer: Can we break this into 1, 2, 3 style steps? Instead of First, Then, Subsequently?

> The following section presents the results of our analyses, each accompanied by a brief description for clarity.

Reviewer: Not sure if this intro is needed.

Author: I removed it

### Heatmaps and Scanpaths

<div style="margin-top: 20px;"></div>

<div class="grid grid-cols-2 gap-4">
    <div class="image-column">
        <img src="./1.phone-heatmap.jpeg" alt="Saliency map over a phone screen" class="image">
    </div>
    <div class="image-column">
        <img src="./2.phone-nadia_scanpath.jpeg" alt="Scanpath over a phone screen" class="image">
    </div>
</div>

<div style="margin-bottom: 20px;"></div>

Two valuable visualisations for understanding gaze behaviour on mobile screens are heatmaps and scanpaths (left and right images, respectively).

Heatmaps visually represent the app areas that receive the most attention, with warmer colours denoting higher fixation rates. These heatmaps can be easily generated within Pupil Cloud using the Reference Image Mapper.

> Meanwhile, scanpaths depict the sequence of eye movements and fixations during visual exploration. The circle size in the scanpath corresponds to fixation duration; larger circles indicate longer user focus on specific screen areas. In the example provided, the focus was primarily on the top panel of the app, as indicated by the largest fixation circle among the three fixations used to generate this scanpath. For those interested in generating this visualisation, our [scanpath tutorial](/scanpath-rim/) provides a step-by-step guide.

Reviewer: Consider omitting "Meanwhile"

Dwell time signifies the total duration of fixation on a specific AOI, serving as an indicator of attentional allocation. Longer dwell times suggest prolonged focus on an AOI. On the other hand, the time to first contact measures how quickly an AOI captures a user's attention, indicating its prominence. 

As shown in this example, there are clear differences between the areas of interest in both attentional allocation and salience that became evident thanks to the quantitative analysis that focused on the measures mentioned earlier. In line with the heatmap and scanpath visualizations, it is evident that dwell time is higher in AOI-1 compared to the other two areas of interest. Additionally, the time to first contact metric revealed that users directed their attention more swiftly towards AOI-0.

For a detailed explanation and visual representation of these metrics, you can refer to our [AOI tutorial](/gaze-metrics-in-aois/), which provides step-by-step instructions and charts for your analysis.

### Map Your Gaze Onto Dynamic Phone Screen Content

<Youtube src="RKrf3YQjzao"/>

> Until now, the tools that we have used have been limited to mapping gaze onto static images. However, the dynamic realm of mobile phones makes it important to study gaze behaviour in a more natural context, particularly during scrolling. We, therefore, used the [dynamic screen mapping tutorial](/map-your-gaze-to-a-2d-screen/) for this purpose. This approach provided a more comprehensive understanding of gaze behaviour in dynamic interfaces.

Reviewer: This is the motivation that I alluded to above.

Upon reviewing the recording, Neon demonstrated good accuracy in capturing gaze behaviour, offering a useful high-level overview of the wearer's focal points.

> Another very useful facet of this tool is its ability to generate a CSV file containing gaze data mapped onto the screen, represented in 2D x, y coordinates. This feature opens avenues for extensive customisation and more in-depth analysis, which we don’t cover here.

Reviewer: Can we make this more concise?

Author: What do you think about something like that: "Another very useful facet of this tool is its ability to generate a CSV file containing gaze data mapped onto the screen, represented in 2D x, y coordinates. Although this feature is beyond the scope of the present article, it opens avenues for extensive customisation and more in-depth analysis, such as creating dynamic heatmaps overlaid on the screen recording."

Reviewer: Is there anywhere we could link out to for this and/or just condense into a single sentence?

Author: This sentence refers to the csv output of the Dynamic RIM article on Alpha Lab. The link to this article has been already added in this section. However, there isn't a dedicated section for the CSV file output of Dynamic RIM in the article, so there's no specific link for that. A more condensed version in one sentence could be: "This tool generates a CSV file with 2D x, y coordinates for gaze data on the screen, enabling extensive customisation and in-depth analysis, though beyond this article's scope." What do you think?

## Let’s Wrap It Up!

> Overall, we were impressed by Neon's capability to effectively characterise viewing behaviour on mobile apps, especially considering its calibration-free nature. The techniques outlined in this tutorial are not confined to our specific application; they hold significant value for various UI/UX and neuromarketing research endeavours. By integrating Neon with the methodologies discussed here, researchers can acquire invaluable insights into user engagement.

Reviewer: Not sure if we need to be the ones to be "impressed". It is indeed impressive that we get such high accuracy, but the reader should be the one to come to that conclusion IMO.

Author: I agree - I modified the sentence: "In this article, we assessed whether Neon is capable of characterizing viewing behaviour on mobile apps. The techniques outlined in this tutorial are not confined to our specific application; they hold significant value for various UI/UX and neuromarketing research endeavours. By integrating Neon with the methodologies discussed here, researchers can acquire invaluable insights into user engagement."

Reviewer: This works.
